### PR TITLE
Connector daily integration tests: new job for alpha connectors

### PR DESCRIPTION
**Pull request by alafanechere**
## What
Declare a separate job to run `alpha` connectors build only in the daily Connector Integration Tests run.
I reverted https://github.com/airbytehq/airbyte/pull/21589 because all the builds can't run in 30mn.
For orchestration and timeout reasons I prefer to declare it in a second job.

### CODE REVIEW

File: .github/workflows/connector_integration_tests.yml [hunk 1]
```diff
@@ -29,3 +29,22 @@ jobs:
         run: python ./tools/bin/ci_integration_workflow_launcher.py base-normalization source-acceptance-test source:alpha source:beta source:GA destination:alpha destination:beta destination:GA
```
Comment (erohmensing) [inline .github/workflows/connector_integration_tests.yml:29]: Hm, the alpha connectors are still in here, maybe you need to merge master in to this branch? Just to validate

File: .github/workflows/connector_integration_tests.yml [hunk 2]
```diff
@@ -29,3 +29,22 @@ jobs:
...
+      - name: Install Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: "zulu"
+          java-version: "17"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install PyYAML requests
+      - name: Launch Integration Tests
```
Comment (erohmensing) [inline .github/workflows/connector_integration_tests.yml:47]:
```suggestion
      - name: Launch Integration Tests (Alpha connectors)
```
Nit: titling the "Launch Integration Tests" job steps for the jobs more clearly

**Review by erohmensing — APPROVED** [review]
I also like having them as 2 separate jobs, in case we end up wanting to point them to different places. Wondering what led us to pick 30 minutes as a timeout for the initial job, and should we have some sort of timeout for this one, even if a longer one?

**Comment by alafanechere** [comment]
> Wondering what led us to pick 30 minutes as a timeout for the initial job.
I don't know exactly but I think it's to have a kind of guarantee that all the test run in 30mn to build the report that is sent on slack afterward in a different action.

> And should we have some sort of timeout for this one, even if a longer one?
I hope the default 360mn is good enough.
The scripts of this workflow triggers /test workflow in parallel, I hope that we can build all our  alpha connectors in parallel in less than 6hours...
I'll figure out tomorrow 😄

**Pull request merged**